solutions/consecutive_substrings.py

```diff
@@ -0,0 +1,62 @@
+def consecutive_substrings(string):
+
+    char_arr = [*string]
```

Comment:
The `string` can be used without splitting it into an array since the string indices would have the same range as the array indices.